pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -47,6 +58,14 @@ buildPythonPackage rec {
     rm -rf ./ml_dtypes
   '';
 
+  disabledTests = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
+    # These tests have expected RuntimeWarning: invalid value encountered in cast
+    # that have not been ignored yet upstream
```

Comment:
what's the upstream issue tracking this? let's make sure to link to it

Reply:
I haven't found the time to research exactly if this is still an upstream issue or our issue. (https://github.com/jax-ml/ml_dtypes/pull/11 in v0.2.0 looks like it's supposed to fix these, but it doesn't.) I will try to find time to do this.

Reply:
yeah let's either fix on our end or report upstream and link to the upstream issue in a code comment